src/constructs/autoscaling/user-data.test.ts

```diff
@@ -99,7 +99,15 @@ describe("GuUserData", () => {
           "Fn::Join": [
             "",
             [
-              "#!/bin/bash\nmkdir -p $(dirname '/etc/testing/secrets.json')\naws s3 cp 's3://test-app-config/secrets.json' '/etc/testing/secrets.json'\nmkdir -p $(dirname '/etc/testing/application.conf')\naws s3 cp 's3://test-app-config/application.conf' '/etc/testing/application.conf'\nmkdir -p $(dirname '/testing/my-app.deb')\naws s3 cp 's3://",
+              "#!/bin/bash\nmkdir -p $(dirname '/etc/testing/secrets.json')\naws s3 cp 's3://",
```

Comment:
Gosh this generated output is hard to visualise 🤕

Reply:
Agreed - this is probably another good argument in support of your conclusion [here](https://github.com/guardian/cdk/pull/298#issuecomment-796525586)!